documentation/dt-bindings.md

```diff
@@ -0,0 +1,55 @@
+## Secure World OP-TEE DT bindings
+
+OP-TEE has a formal DT binding at
+
+```
+	firmware {
+		optee {
+		};
+	};
+```
+
+The Linux kernel device tree bindings documentation
+explains the properties that may appear there that
+are of interest to Normal World.
+
+OP-TEE may also be passed a DTB at boot-time, which
+may be different to, or modified from, the DTB passed
+to Normal World.
```

Comment:
No. Commit 76358becffb9 ("core: arm: generic boot: edit device tree") assumes that modifications  made by OP-TEE to the DT will be visible to the kernel. So, for all practical purpose, one should consider that there is only one DT, which might be modified by OP-TEE before it is passed to the NW kernel. Although the actual implementation might be different (for instance the firmware might make copy the DT between secure and non-secure memory).